fetch/main.js

```diff
@@ -0,0 +1,14 @@
+async function getData(url) {
+  try {
+    const response = await fetch(url);
+    if (!response.ok) {
+      throw new Error('Network response not okay...');
+    }
+    const data = await response.json();
+    console.log(data);
+  } catch (err) {
+    console.error(`An unexpected error occurred...: ${err}`);
```

Comment:
Log the actual error so you get the stack trace for debugging purposes. If you interpolate it like this you only log `err.toString()`, which doesn't include the stack trace.